src/platform/site-wide/sass/style.scss

```diff
@@ -4,7 +4,7 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
 @import "~@department-of-veterans-affairs/formation/sass/shared-variables";
 @import "~@department-of-veterans-affairs/formation/sass/core";
 
-@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/utilities";
+@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/utilities.css";
```

Comment:
I've further isolated the break to this line specifically.

Reply:
Cc: @jamigibbs

Reply:
Removing the `.css` allowed the Sass file to load the stylesheets in the order they are listed in the file instead of "hoisting" the utilities to the top when it compiled which was causing specificity issues. 

What I suspect is happening is that (and similar to the [issue discussed here](https://dsva.slack.com/archives/CBU0KDSB1/p1727372949849339)) is that the "old" breakpoint names are not loading properly and they need to be updated to the new USWD names. This isn't what we intended to happen and I'm not sure why it is but the solution at this point is probably what is being done in this [PR here that's updating flex](https://github.com/department-of-veterans-affairs/vets-website/pull/32120) utilities.